docs/content/reference/cli/authelia/authelia_debug_tls.md

```
@@ -0,0 +1,53 @@
+
+## authelia debug tls
+
+Perform a TLS debug operation
+
+### Synopsis
+
+Perform a TLS debug operation.
+
+This subcommand allows checking a remote server's TLS configuration and the ability to validate the certificate.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)